contributors/design-proposals/api-machinery/aggregated-api-servers.md

```diff
@@ -262,7 +262,7 @@ There were other alternatives that we had discussed.
 
 * Validate servers: We should have some conformance tests that validate that the
   servers follow kubernetes api-conventions.
-* Provide centralised auth service: It is very hard to ensure homogeneous auth
+* Provide centralized auth service: It is very hard to ensure homogeneous auth
```

Comment:
heh.  Anti-british?

Reply:
Yeah, the UK spelling is fine. We can remove this change :)

Reply:
thanks a lot.